support/azure/virtual-machines/windows/support-agent-extensions.md

```diff
@@ -46,7 +46,6 @@ The VM agent consists of three services that must be running:
 
 - RDAgent
 - Windows Azure Guest Agent
-- Microsoft Azure Telemetry Service
 
 Make sure that these three services are running, and then check the extensions installation and startup.
```

Comment:
Make sure that these services are running and then check the extensions installation and startup.

Reply:
Hello Jarrett,

The Microsoft Azure Telemetry service is integrated with the RDagent service. When a new machine is deployed, only these two services are available:

For more details, you can refer to this link:  
https://learn.microsoft.com/en-us/troubleshoot/azure/virtual-machines/windows/windows-azure-guest-agent